workspaces/src/network/account.rs

```diff
@@ -17,6 +24,60 @@ impl Account {
     pub(crate) fn signer(&self) -> &InMemorySigner {
         &self.signer
     }
+
+    /// Call a contract on the network specified within `worker`, and return
+    /// a Builder object that we will make use to populate the rest of the
+    /// call details.
+    pub fn call<'a, T: Network>(
+        &self,
+        worker: &'a Worker<T>,
+        contract_id: AccountId,
+        function: String,
```

Comment:
I'll change the function signature since that would also be easier on the user to specify static strings, but underneath, it still needs to be an owned string due to how `near_primitives::transaction::FunctionCallAction` requires an owned string as an argument